README.markdown

```diff
@@ -28,11 +28,14 @@ then simply copy and paste:
   the first file in a brand new Ruby project might very well be derived from
   your `.irbrc`.  I consider this a feature.
 * If your file is consistently indented with hard tabs, `'shiftwidth'` will be
-  set to your `'tabstop'`.  Otherwise, a `'tabstop'` of 8 is enforced.
+  set to your `'tabstop'`.  Otherwise, a `'tabstop'` of 8 unless configured.
```

Comment:
This does not clearly explain what happens.
How about 'Otherwise, a default `tabstop` (8 if not re-configured during install) is enforced.'?